actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -106,7 +106,7 @@ def tag_option(key, value, escape)
         private
           def prefix_tag_option(prefix, key, value, escape)
             key = "#{prefix}-#{key.to_s.dasherize}"
-            unless value.is_a?(String) || value.is_a?(Symbol) || value.is_a?(BigDecimal)
+            unless  (prefix.to_s == "aria" && value.is_a?(Array)) || value.is_a?(String) || value.is_a?(Symbol) || value.is_a?(BigDecimal)
```

Comment:
This feels very specific and inelegant, but I wasn't able to find a better spot to put the special casing. I'm open to suggestions.

Reply:
I think the best way to implement this is to change the code at `tag_options` to threat `aria` differently of `data`. Maybe `aria` should not be treated as a prefix tag in the `TAG_TYPES` hash.